src/consensus/role_timeout.rs

```diff
@@ -66,22 +66,21 @@ impl TimeoutState {
 pub(super) struct TimeoutRoleState {
     pub(super) requests: HashSet<SignedByValidator<ConsensusNetMessage>>,
     pub(super) state: TimeoutState,
-    pub(super) highest_seen_prepare_qc: Option<(Slot, View, QuorumCertificate)>,
+    pub(super) highest_seen_prepare_qc: Option<(Slot, QuorumCertificate)>,
```

Comment:
i guess should be renamed with smotheling like
```suggestion
    pub(super) latest_seen_prepare_qc: Option<(Slot, QuorumCertificate)>,
```

Reply:
Je trouve ça plus ambiguë en vrai. Highest parle du slot, latest c'est temporel.